src/docbuilder/rustwide_builder.rs

```diff
@@ -1019,4 +1028,19 @@ mod tests {
             Ok(())
         });
     }
+
+    #[test]
+    #[ignore]
+    fn test_locked_fails_unlocked_needs_new_deps() {
```

Comment:
This is a very slow test, it takes ~2 minutes on my machine, there's probably smaller crates that could be used instead, but I don't know an easy way to find them.

Reply:
We could just publish one - if we prefix it with `docs_rs_test` I don't think anyone will mind.

Reply:
Mmm yeah, that should be pretty easy to setup. I'll publish one and update this test tomorrow.

Reply:
(we could also publish to staging.crates.io, but I don't think the 1-10 crates we ever end up using are a big deal compared to the ~35k that already exist)